--- modules/paths_internal.py
+++ modules/paths_internal.py
@@ -32,5 +32,5 @@ data_path = cmd_opts_pre.data_dir
 models_path = cmd_opts_pre.models_dir if cmd_opts_pre.models_dir else os.path.join(data_path, "models")
 extensions_dir = os.path.join(data_path, "extensions")
 extensions_builtin_dir = os.path.join(script_path, "extensions-builtin")
-config_states_dir = os.path.join(script_path, "config_states")
+config_states_dir = os.path.join(data_path, "config_states")
 default_output_dir = os.path.join(data_path, "outputs")

 roboto_ttf_file = os.path.join(modules_path, 'Roboto-Regular.ttf')
